src/lib/bootstrap.js

```diff
@@ -89,7 +89,10 @@ function startVerdaccio(config: any,
       } else { // http
         webServer = http.createServer(app);
       }
-
+      if (config.server && config.server.keepAliveTimeout) {
+        // $FlowFixMe library definition for node is not up to date (doesn't contain recent 8.0 changes)
+        webServer.keepAliveTimeout = config.server.keepAliveTimeout;
```

Comment:
This PR sets the timeout in seconds, while the [node docs](https://nodejs.org/api/http.html#http_server_keepalivetimeout) suggest that the value is in microseconds.

```suggestion
        webServer.keepAliveTimeout = config.server.keepAliveTimeout * 1000;
```

Reply:
Good catch, could you PR?

Reply:
yes, on it

Reply:
#1221